api/client.go

```diff
@@ -207,6 +207,25 @@ func (c *Client) doRequestRaw(request *http.Request) (*http.Response, error) {
 	return response, nil
 }
 
+func (c *Client) doRequest(request *http.Request) (bool, []byte, error) {
+	response, err := c.doRequestRaw(request)
+	if err != nil {
+		return false, nil, err
+	}
+	defer response.Body.Close()
+
+	body, err := ioutil.ReadAll(response.Body)
```

Comment:
```suggestion
	body, err := io.ReadAll(response.Body)
```

`ioutil` is deprecated. Lets not add new usages, although its probably used elsewhere in the CLI. Lets fix those in a separate PR.